### PR TITLE
fix: add SamirPaul1

### LINKED ISSUE #936: Samir Paul
## Profile Screenshot

SamirPaul1-Overview

https://raw.githubusercontent.com/SamirPaul1/cv/main/assets/img/SamirPaul1-Overview.png

### PR DESCRIPTION
**Pull request by SamirPaul1**
resolves #936

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/SamirPaul1.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://raw.githubusercontent.com/SamirPaul1/cv/main/assets/img/SamirPaul1-Overview.png",
```
Comment (Panquesito7) [inline profiles/SamirPaul1.json:2]: I recommend more of an image like this.

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/147397131-a108815e-874b-455c-ba15-224c954b44ca.png",
```

File: profiles/SamirPaul1.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "issueId": 936,
+  "name": "SAMIR PAUL",
```
Comment (Panquesito7) [inline profiles/SamirPaul1.json:4]:
```suggestion
  "name": "Samir Paul",
```

**Review by krshdev — CHANGES_REQUESTED** [review]
Hey @SamirPaul1 👋 Do you need any help with this? Please do the changes requested above by @Panquesito7 🙂

**Comment by SamirPaul1** [comment]
Hello @krshdev I have changed the SamirPaul1.json file now you can merge it.

**Review by krshdev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**